src/PollingBlockTracker.ts

```diff
@@ -85,10 +85,11 @@ export class PollingBlockTracker extends BaseBlockTracker {
             err.stack ?? err
           }`,
         );
+        console.error(newErr);
```

Comment:
Hmm, this seems like a regression. Currently we can attach an `error` handler to capture these errors handle handle them in some custom way without it being logged to the console. This change makes the console error impossible to prevent.

Reply:
Ah, okay. I changed this because it ended up simplifying the tests, which are extremely hard to work with. Your point is valid though. I can change this back.